models/models.py

```diff
@@ -60,6 +60,37 @@ def forward(self, imgs):
         return out
 
 
+class YOLOv8Builder(ModelBuilder):
+    def forward(self, imgs):
+        out = []
+        batch_size = imgs.shape[0]
+        print(batch_size)
+        #x = imgs.resize_(batch_size, 3, 640, 640)
+        x = preprocess_for_YOLO(imgs[0], device=self.device)
+        #shape1 = x.shape[2:]
+        self.count += 1
+        self.in_block.to(self.device)
+        self.backbone.to(self.device)
+        self.out_block.to(self.device)
+        #x = self.in_block(x)
+        print(x.max())
```

Comment:
Is this class necessary? If possible, try to work around without it. Also, remove all `print`

Reply:
Check out the model I have for plates in `models.py`, it might be possible to do something similar